### ISSUE TITLE
Not working on Safari IOS 12.1.3

### ISSUE THREAD

**Issue by mark-wff** [schmich/instascan#204]
I can't see a Camera Image on the Demo-Website.

**Comment by kibagami-jubei** [comment]
I would say stay away if you're having to use this on iOS. Seems like it works fine on Chrome or anything else non-apple.

**Comment by kibagami-jubei** [comment]
Also, there are ways to force this to work on apple iOS, just look around this thread.

**Comment by mark-wff** [comment]
Yes, found a solution for IOS in another ticket. But not merged jet....

The issue can be closed.

**Comment by vgflutter** [comment]
What is this ticket?

**Comment by mark-wff** [comment]
@feof The Demo Website/ The Code don't work on Safari IOS 12.1.3

**Comment by auriga-dev** [comment]
I have the same issue doesn't work on 12.1.2
Any idea how to solve the issue?
Thanks

**Comment by mark-wff** [comment]
That worked for me: https://github.com/schmich/instascan/issues/182#issuecomment-443388022
But no solution merged in Master. And with this solution, I'm not able to switch between cameras.

**Comment by fagianijunior** [comment]
In safari and Chrome 71.0.3578.98 in MacOs does not show camera too

**Comment by spacehunter** [comment]
Works in Chrome Version 72.0.3626.121

**Comment by USOTT** [comment]
shaa2
